src/software/thunderscope/replay/replay_file_debugging_script.py

```diff
@@ -52,7 +51,11 @@ def read_one_chunk(replay_file_name):
             #######################################
             # Do something with the protobuf here #
             #######################################
-            print(f"{line_num}: {float(timestamp)}: {protobuf_type} - {proto=}")
+            print(
+                "{}: {}: {} - {=}".format(
+                    line_num, float(timestamp), protobuf_type, proto
+                )
+            )
```

Comment:
Personally I find f-strings easier to read since the variable is used within the string. Or was this an automatic change made by the formatting script?

Reply:
That was intentional. F string was too long and the auto format seemed to failed as a result of it.

Reply:
The auto formatter cannot guarantee the validity of the program with such a long f string.

Reply:
Interesting. I added a comment to #3175 to update our Black formatter as well. Newer versions should handle f-strings better.